Database/QuestieDB.lua

```diff
@@ -790,14 +790,16 @@ function QuestieDB:GetQuest(questId) -- /dump QuestieDB:GetQuest(867)
             }
             tinsert(QO.ObjectiveData, reputationObjective);
         end
-        if objectives[5] then
-            local killCreditObjective = {
-                Type = "killcredit",
-                IdList = objectives[5][1],
-                RootId = objectives[5][2],
-                Text = objectives[5][3]
-            }
-            tinsert(QO.ObjectiveData, killCreditObjective);
+        if type(objectives[5]) == "table" and #objectives[5] > 0 then
```

Comment:
We can assume this is always `nil` or a table, Can't we?

```suggestion
        if objectives[5] and #objectives[5] > 0 then
```

Reply:
To be honest this was kinda cargo-culted over from the compiler and is probably not needed *here*.

Reply:
I totally agree, this (and also the other point) is nothing worth investing much time at this point. Especially if the code is working as it is.